docs/1.0-release-notes.md

```diff
@@ -0,0 +1,248 @@
+This substantially reduces the number of updates processed by Contour.
+
+Fixes #1372
+
+### Contour filters Endpoint updates
+
+Contour now supports filtering update notifications in some circumstances. Specifically Envoy's EDS watches will no longer fire unless the specific EDS entry requested is updated. This should significantly reduce the number of spurious EDS updates send to Envoy.
+
+Updates #426, #499
+
+### Minor improvements
+
+- The `contour` binary now executes a graceful shutdown when sent SIGTERM. Thanks @alexbrand. Fixes #1364.
+- Contour now preserves the `X-Request-Id` header if present. Fixes #1509.
+- Contour's quickstart documentation now references the current stable version of Contour. Fixes #952.
+- Contour will no longer present a secret via SDS if that secret is not referenced by a valid virtualhost. #1165
+- The `envoyproxy/go-control-plane` package has nbeen upgraded to version 0.9.0. `go-control-plane` 0.9.0 switches to the `google/protobuf` library which results in a 4mb smaller binary. Neat.
+- Our `CONTRIBUTING` documentation has been updated to encourage contributors to squash their commits. Thanks @stevesloka.
+- The markup of several of our pages has been corrected to render properly on GitHub. Thanks @sudeeptoroy.
+- Envoy's `/healthz` endpoint has been replaced with `/ready` for Pod readiness. Fixes #1277. Thanks @rochacon.
+- IngressRoute objects now forbid `*` anywhere in the `spec.virtualhost.fqdn` field. Fixes #1234.
+- Contour is built with Go 1.13.1.
+- `make help` target added. Thanks @jpeach.
+- `prefix` conditions must start with a slash. Fixes #1628. Thanks @youngnick.
+- Duplicate HTTPProxy `header` conditions are now rejected. Fixes #1559. Thanks @youngnick.
+- HTTPProxy `route` or `include` blocks with more than one `prefix` condition are now rejected. Fixes #1611. Thanks @stevesloka.
+- The `X-Request-Id` header is now no longer removed from incoming requests. Fixes #1487.
+- `HTTPProxy` `include`s no longer require a `namespace` key. If no `namespace` is provided, the included HTTPProxy is inferred to be in the same namespace as its parent. Fixes #1574. Thanks @youngnick.
+- The ability to write the bootstrap configuration to standard out via `contour bootstrap -- -` has been added. Thanks @jpeach.
+- Contour now validates that TLS certificates either bare the type `kubernetes.io/tls` or, in the case of upstream validation certificates, contain a non empty `ca.crt` key. Fixes #1697. Thanks @jpeach.
+- `x_trace_id` has been added to the set of JSON loggable fields. Fixes #1734. Thanks @cw-sakamoto!
+- Obsolute Heptio branding has been removed from `contour cli`. Thanks @jpeach.
+- Contour is built with Go 1.13.3.
+
+## Bug fixes
+
+### TLS certificate validation improvements
+
+Contour 1.0.0-rc.2 improves the TLS certificate validation added in rc.1. Contour is now less likely to reject valid certificates that contain unexpected elliptic curve parameters.
```

Comment:
Should this reference the 1.0.0 release rather than the rc?

Reply:
I dropped this section as it refers to something that happened during the rc process.